pkg/apis/eventing/v1alpha1/channel_types.go

```diff
@@ -78,8 +78,19 @@ type ChannelSpec struct {
 	// +optional
 	Arguments *runtime.RawExtension `json:"arguments,omitempty"`
 
-	// Channel conforms to Duck type Channelable.
-	Channelable *duckv1alpha1.Channelable `json:"channelable,omitempty"`
+	// Subscribers is information about subscriptions used to implement message forwarding.
+	Subscribers []ChannelSubscriberSpec `json:"subscribers,omitempty"`
```

Comment:
I think it is more flexible to not use the duck type specs inside our own specs, so I have duplicated what the duck type is expecting to find. This decouples them but we still get the validation that that match from the type inspection runtime check.